README.md

```
@@ -1 +1,179 @@
+echo "JAVA_HOME=${JAVA_HOME}"
+echo "MAVEN_HOME=${MAVEN_HOME}"
+
+# Clean up unnecessary packages
+echo "Cleaning up unnecessary packages..."
+sudo apt autoremove -y
+
+echo "Java and Maven installation completed successfully!"
+
+mvn clean install : it will build the app
+
+mvn spring-boot:run : this would run the jar file
```

Markdownlint (0.37.0)

Expected: atx; Actual: setext
Heading style

(MD003, heading-style)